osclib/stagingapi.py

```diff
@@ -936,12 +936,16 @@ def get_sub_packages(self, package, project):
         # guarantee the sub-pacakges are created according to the
         # specfiles of main package. Therefore, main package must be
         # created before through get_sub_packages().
-        filelist = self.get_filelist_for_package(pkgname=package, project=project, expand='1', extension='spec')
+        filelist = self.get_filelist_for_package(pkgname=package, project=project, expand='1')
+        if '_multibuild' in filelist:
```

Comment:
I suppose `_multibuild` file should be filtered out in `get_sub_packages()` instead of return it back?